main/test/flix/Test.Dec.Effect.flix

```diff
@@ -0,0 +1,14 @@
+namespace Test/Dec/Effect {
+    eff Console {
+        // Will be removed after Weeder checks are added
+        def readLine(): String
+        def println(s: String): Unit
+    }
+
+    eff Empty
+
+    // Will be removed after Weeder checks are added
+    eff Polymorphic[a] {
+        def op[a](): a
+    }
```

Comment:
I think you can already a bunch more of tests :)

Reply:
Were there other particular vectors of possible error you were looking for, or should I just reproduce a bunch of the standard effects from literature?

Reply:
Not specifically, just some systematic testing.